src/lib/gundb/UserStorageClass.js

```diff
@@ -1020,10 +1019,10 @@ export class UserStorage {
       attr = 'username'
     } else if (this.wallet.wallet.utils.isAddress(field)) {
       return field
-    }
-    if (attr === undefined) {
+    } else {
       return undefined
     }
+
     const value = UserStorage.cleanFieldForIndex(attr, field)
```

Comment:
The lines bellow will never be executed right? If this is ok shouldn't we just remove those?

Reply:
why? if `field` happens to be mobile, email or username... it will execute those lines, isn't it?

Reply:
oh, that's right! I didn't realize that you didn't change t this in tihs PR so this is ok.
What seems weird to me that some conditions are returning a value and some others don't. Would be easier to read if we had the ones that return as guards in the beginning and then the other conditions.

Reply:
Ok, fixed after what we discussed offline.Thanks!